Examples/DemoARViewController.swift

```diff
@@ -81,8 +82,8 @@ final class DemoARViewController: UIViewController, ARSCNViewDelegate, ARSession
 
     private func insert(on plane: SCNNode, from hitResult: ARHitTestResult) {
         //Set up initial terrain and materials
-        let terrainNode = TerrainNode(minLat: 50.044660402821592, maxLat: 50.120873988090956,
-                                      minLon: -122.99017089272466, maxLon: -122.86824490727534)
+        let terrainNode = TerrainNode(minLat: 53.394747374316, maxLat: 53.422495172318,
```

Comment:
did you want to change these?

Reply:
Also maybe add a comment about the general location of these coordinates